psutil/_pslinux.py

```diff
@@ -1229,6 +1229,49 @@ def sensors_temperatures():
 
         ret[unit_name].append((label, current, high, critical))
 
+    # Indication that no sensors were detected in /sys/class/hwmon/
+    if len(basenames) == 0:
```

Comment:
You can simply spell it as `if not basenames:` instead

Reply:
`basenames = sorted(set([x.split('_')[0] for x in basenames]))` returns an empty list even if not files match, so `None` will not enter the `if`